pkg/api/batch/operation.go

```diff
@@ -52,9 +52,9 @@ type Operation struct {
 	RecoveryRevealValue string `json:"recoveryRevealValue"`
 
 	// Hash of reveal value for the next update operation
-	NextUpdateCommitmentHash string `json:"nextUpdateCommitmentHash"`
+	UpdateCommitment string `json:"updateCommitment"`
 	// Hash of reveal value for next recovery/deactivate operation
-	NextRecoveryCommitmentHash string `json:"nextRecoveryCommitmentHash"`
+	RecoveryCommitment string `json:"recoveryCommitment"`
```

Comment:
underscores

Reply:
Operation is our internal model - so we can use whatever we want.

Reply:
Prefer to be consistent. I'll open a fresh issue for the internal model.

Reply:
https://github.com/trustbloc/sidetree-core-go/issues/221